packages/components/src/button/style.scss

```diff
@@ -9,24 +9,27 @@
 	background: none;
 	transition: box-shadow 0.1s linear;
 	@include reduce-motion("transition");
-
-	&.is-button {
-		padding: 0 10px;
-		line-height: 2;
-		height: 28px;
-		border-radius: 3px;
-		white-space: nowrap;
-		border-width: $border-width;
-		border-style: solid;
+	height: 36px;
+	align-items: center;
+	box-sizing: border-box;
+	padding: 0 8px;
+	overflow: hidden;
```

Comment:
hey @youknowriad 
currently, I am trying to make some progress on: https://github.com/WordPress/gutenberg/pull/15830
this `overflow: hidden;` is causing some issues:
before

![image](https://user-images.githubusercontent.com/1477453/77152367-7f396180-6aa0-11ea-98ed-4efc870a50e3.png)

***
after

![image](https://user-images.githubusercontent.com/1477453/77152374-882a3300-6aa0-11ea-844c-fe2bc285c977.png)

Reply:
can you please help me understand what issue was fixing this `overflow: hidden;`?
or maybe you have a solution for those cut labels?

Reply:
I'm not really certain it's needed. I believe this PR just copied it from the icon button stylesheet.

Reply:
This is also causing an issue with a block directory button. @nicolad Did you reach any solution on this?